source/ContractConfigurator/Parameter/VesselParameter/Rendezvous.cs

```diff
@@ -167,17 +167,14 @@ protected override void OnUpdate()
                             if (distance < this.distance)
                             {
                                 rendezvous = true;
-                                forceStateChange |= SetState(v1, ParameterState.Complete);
-                                forceStateChange |= SetState(v, ParameterState.Complete);
                             }
+                            forceStateChange |= SetState(v1, rendezvous ? ParameterState.Complete : ParameterState.Incomplete);
```

Comment:
If there's more than 1 other vessel loaded then you are now setting the state to Complete for the rest as well if the first one in the loop satisfies the distance condition.